### PR TITLE
Use triple-quotes in docstrings

### PR DESCRIPTION
**Pull request by atombrella**
I'm not sure if this is a priority, however, it's addressed in PEP8 with PEP257. There were quite a lot of them.

It was quick to find (and fix) them with PyCharm, so I won't be discouraged if the PR is closed without being merged.

### CODE REVIEW

File: django/utils/dateformat.py [hunk 1]
```diff
@@ -186,8 +186,9 @@ def T(self):
             name = self.format('O')
         return six.text_type(name)
 
+    @property
```
Comment (timgraham) [inline django/utils/dateformat.py]: Some unrelated changes seem to have crept in.

**Comment by timgraham** [comment]
I guess I don't have a strong feeling one way or another, however, if accepted, it would be nice to have some linting to enforce a particular style. Without that, it's a bit difficult to enforce a certain style.

**Comment by atombrella** [comment]
In the documentation, https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/ PEP257 is addressed. I'll update the PR so tests pass, and the suggestion about using a lint rule is good.

**Comment by cjerdonek** [comment]
> In the documentation, https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/ PEP257 is addressed.

FYI, the coding style docs say to do this:

> In docstrings, follow PEP 257. For example:

    def foo():
        """
        Calculate something and return the result.
        """
        ...

But the latest patch changes many docstrings to be like this:

    def foo():
        """Calculate something and return the result."""
        ...

**Comment by timgraham** [comment]
All the changes I see are changing "Calculate something and return the result." to """Calculate something and return the result.""". I think it's fine rather than changing to multiline docstrings and introducing a lot more lines.

**Comment by atombrella** [comment]
I've found docformatter which should format comments according to PEP257. I tried with a single file, and comments are changed from:

        """
        By default, call_command should not trigger the check framework, unless
        specifically asked.
        """

to something like:

        """By default, call_command should not trigger the check framework,
        unless specifically asked."""

There's a PEP257 which warns about everything it can think of (missing dots at the end of comments, violation of not using an imperative form of a verb as the first in the docstring etc.). I think the convention of using three double-quotes instead of various combinations of either one, a single quote or three single-quotes should be enforced.

**Comment by timgraham** [comment]
I lean toward abandoning this since the current style hasn't caused much problems. We clean up a little as we go. I remember a past discussion about doc linters where the conclusion was that it just wasn't worth the effort.
